uvm/servlets/admin/app/overrides/form/field/VTypes.js

```diff
@@ -243,11 +243,11 @@ Ext.define('Ung.overrides.form.field.VTypes', {
         case 'any':
             return true;
         default:
-            if (val.indexOf('>') !== -1 && val.indexOf(',') === -1) {
-                return this.isSinglePortValid( val.substring( val.indexOf('>') + 1 ));
+            if (val.indexOf('>=') !== -1 && val.indexOf(',') === -1) {
```

Comment:
The val check for >=, >. <, and <= are all values the user is able to specify so we need to support those as well.  I think this code could break that, so be sure to retain support for the "non =" operations as well.

Reply:
Added code to handle `>` and `<` Symbols

![Screenshot from 2024-02-08 20-42-01](https://github.com/untangle/ngfw_src/assets/154422821/8b13a42c-e90d-4deb-badf-9f9727bd080d)

![Screenshot from 2024-02-08 20-41-42](https://github.com/untangle/ngfw_src/assets/154422821/7f1e93df-2339-4f26-9be3-c76fa4507be2)

![Screenshot from 2024-02-08 20-41-06](https://github.com/untangle/ngfw_src/assets/154422821/0840196d-c0aa-4e92-853b-530f5d1bf232)